src/styles/blocks/main.scss

```diff
@@ -0,0 +1,22 @@
+.main {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+
+  grid-auto-flow: row;
+  gap: 46px 48px;
+
+  justify-content: center;
+
+  @include on-screen2 {
```

Comment:
The mixin `on-screen2` is used but not defined. Please define this mixin to ensure the responsive design for two columns is applied correctly.